src/deploy/misc/00-sandbox.ts

```diff
@@ -5,8 +5,7 @@ import { DeployFunction } from "hardhat-deploy/types";
 /** Deployment stub */
 const deploy: DeployFunction = async () => {
```

Comment:
If this is should have parts commented or uncommented as necessary it may be good to add this file to gitignore so it says committed but changes aren't tracked.

Reply:
I'll get rid of these "sandboxes", it is more of a personal workflow in the end

Reply:
added some general util instead to get common deployment context and logging for hardhat context